spread.yaml

```diff
@@ -729,6 +729,54 @@ backends:
             - amazon-linux-2023-64:
                   username: root
                   password: root
+    garden-nested:
+        type: adhoc
+        allocate: |
```

Comment:
the allocation seems mostly identical to the non nested one, can you perhaps use anchor to reference the script we already have there?